flink/src/test/java/org/apache/iceberg/flink/data/TestFlinkParquetReader.java

```diff
@@ -35,51 +36,49 @@
 
 import static org.apache.iceberg.flink.data.RandomData.COMPLEX_SCHEMA;
 
-public class TestFlinkParquetReaderWriter {
+public class TestFlinkParquetReader {
```

Comment:
Better to extend the `org.apache.iceberg.data.DataTest` because it provides more test cases, which `COMPLEX_SCHEMA`  did not cover.   ( I used `COMPLEX_SCHEMA` before,  because I don't know there's a better testing method).

Reply:
Make sense to me and updated.